.circleci/config.yml

```diff
@@ -119,3 +182,4 @@ workflows:
   recipes:
     jobs:
       - build_and_test
+      # - windows_test
```

Comment:
Is there a reason for not enabling this?

Reply:
The 30_get-pipenv test currently fails for windows.  I plan to enable once that's working.